js&css/web-accessible/www.youtube.com/player.js

```diff
@@ -664,6 +664,40 @@ ImprovedTube.playerRotateButton = function () {
 		this.elements.buttons['it-rotate-styles'].remove();
 	}
 };
+
+/*------------------------------------------------------------------------------
+FIT-TO-WIN BUTTON
+------------------------------------------------------------------------------*/
+ImprovedTube.playerFitToWinButton = function () {
+	if (this.storage.player_fit_to_win_button === true && (/watch\?/.test(location.href))) {
+		const svgMarkup = `<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" id="ftw-icon">
+		<path d="M21 3 9 15"/><path d="M12 3H3v18h18v-9"/><path d="M16 3h5v5"/><path d="M14 15H9v-5"/></svg>`;
+
+    let tempContainer = document.createElement("div");
+    tempContainer.innerHTML = svgMarkup;
+    const svg = tempContainer.firstChild;
+		this.createPlayerButton({
+			id: 'it-fit-to-win-player-button',
+			child: svg,
+			opacity: 0.85,
+			position: "right",
+			onclick: function () {
+				let previousSize = ImprovedTube.storage.player_size === "fit_to_window" ? "do_not_change" : ImprovedTube.storage.player_size;
```

Comment:
I didn't any bug you described until I remembered the CSS trick we added while working on the `Comments -> Sidebar` feature #1681. Basically "do_not_change" doesn't get applied by default, only when users change the Player Size at least one time. I should add a nullish operator here: 
`let previousSize = ImprovedTube.storage.player_size === "fit_to_window" ? "do_not_change" : (ImprovedTube.storage.player_size ?? "do_not_change")`

Reply:
thanks, oh, right - and we / somebody can work on that (https://github.com/code-charity/youtube/issues/1685)